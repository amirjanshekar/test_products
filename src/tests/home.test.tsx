import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { describe, expect, it } from "@jest/globals";
import React from "react";
import HomePage from "@/app/page";

describe("Home", () => {
  it("renders homepage unchanged", async () => {
    const { container } = await render(<HomePage />);
    await expect(container).toMatchSnapshot();
  });
});
