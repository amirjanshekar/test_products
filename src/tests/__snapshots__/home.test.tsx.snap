// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Home renders homepage unchanged 1`] = `<div />`;
